migrate-add-videos.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE USER ADD COLUMN VIDEOS VARCHAR(255);
```

Comment:
Actually, sometimes URLs are huge and with the comma separated approach 255 might not b enough in some cases

Reply:
Made it VARCHAR(1024)

Reply:
@I-Mircheva why do we need this SQL ? The schema should be updated automatically anyway.